docs/source/release-history.rst

```diff
@@ -0,0 +1,10 @@
+***************
+Release History
+***************
+
+v0.0.10 (2019-06-04)
```

Comment:
Date needs to be updated once you are ready to actually tag. Otherwise looks good!